definitions/procedures/foreman_docker/remove_foreman_docker.rb

```diff
@@ -6,11 +6,12 @@ class RemoveForemanDocker < ForemanMaintain::Procedure
     end
 
     def docker_package
-      'tfm-rubygem-foreman_docker'
+      find_package(foreman_plugin_name('foreman_docker'))
     end
 
     def run
       return unless execute?("rpm -q #{docker_package}")
```

Comment:
Oh, and this should probably use the package manager's `installed?` method instead.

Reply:
removed this and moved condition to confine block